### DISCUSSION TITLE
TraDOGE v1.3 for Windows

### DISCUSSION THREAD

**Discussion by gschurck** [gschurck/tradoge#21] [Show and tell]
## Bugs fixes and optimizations, add error loggin

- **Fixed bugs**
  - Fixed bug `NoneType` when your Binance account was empty
  - Fixed bug with tweets retrieval not working
  - Optimized imports
  - Reduced bundle size
- **Error logging**
  - If programs craches, full error message is saved in `error.log` file

<hr /><em>This discussion was created from the release <a href='https://github.com/gschurck/tradoge/releases/tag/v1.3.1'>TraDOGE v1.3.1 for Windows</a>.</em>

**Comment by gschurck** [comment]
Hello, I'm working again on the project because I have free time again. I fixed some big bugs, sorry for the delay.
I am also working to implement Futures and leverage with TraDOGE !

**Comment by NismoGTR46** [comment]
Hello gschurck and ty for your work on this project!
I had it running in windows for a week but now it seems to load the script and close it instantly.
What could be the problem?

**Reply by gschurck** [reply]
Hello, thanks !
Can you check if there is something in the logs file ?

**Comment by NismoGTR46** [comment]
CRITICAL:Error log:Fatal error
Traceback (most recent call last):
  File "tradoge.py", line 594, in <module>
  File "tradoge.py", line 487, in main
IndexError: list index out of range

This is what i'm getting in the errorlog

Op vr 4 feb. 2022 om 10:58 schreef Guillaume Schurck <
***@***.***>:

**Reply by gschurck** [reply]
Ok thanks, found the issue.
![image](https://user-images.githubusercontent.com/21091232/152564665-ef044061-3831-4d6a-a502-e16df493e56d.png)
I had the same error, and it's due to the twint library to search tweets which is totally broken sadly and not updated.
I'm currently working on a new version to implement Twitter API directly and avoid these issues.
Unfortunately it's unusable now.
I hope I will be able push this update by the end of next week.

**Reply by gschurck** [reply]
Also next time use the [Issues page](https://github.com/gschurck/tradoge/issues) to ask for help when you have an error please, thanks.

**Comment by NismoGTR46** [comment]
Ty very much mate for your work!
If i can make a profit from it i will send you some profits!

ty and enjoy your weekend

Op vr 4 feb. 2022 om 17:25 schreef Guillaume Schurck <
***@***.***>:

**Reply by gschurck** [reply]
Haha thanks !
Btw for the next update I'm also working on Binance Futures, mainly to add leverage.

**Comment by NismoGTR46** [comment]
Hey Guillaume,

Is there an update for the script atm.

grtz jens

Op vr 4 feb. 2022 om 17:46 schreef Guillaume Schurck <
***@***.***>:

**Reply by gschurck** [reply]
Hey, I am (literally) currently working on it.

**Comment by NismoGTR46** [comment]
yes ty very much! like told if it works i'm not greedy and will give some
for your work!

grtz

Op do 17 feb. 2022 om 00:13 schreef Guillaume Schurck <
***@***.***>:
